net/hysteria/Makefile

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2023 Findlay Feng <i@fengch.me>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hysteria
+PKG_VERSION:=2.5.0
+PKG_RELEASE:=1
+
+GO_PKG:=github.com/apernet/hysteria
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://$(GO_PKG).git
+PKG_MIRROR_HASH:=98c4e776dc0c14de820a7113dc39a1cbe4c2ba07b203852bd2a7a65607f93fed
+PKG_SOURCE_VERSION:=app/v$(PKG_VERSION)
+
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Findlay Feng <i@fengch.me>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/golang/golang-values.mk
```

Comment:
should be 
```include ../../lang/golang/golang-package.mk```

Reply:
Makefile:34: /golang-package.mk: No such file or directory
make[2]: *** No rule to make target '/golang-package.mk'.  Stop.

```
Package: hysteria-clientd
Submenu: Web Servers/Proxies
Version: 2.5.0-r1
Depends: +libc hysteria +hysteria-configs +kmod-nft-tproxy +kmod-nft-socket +dnsmasq-full
Conflicts:
Menu-Depends:
Provides:
Section: net
Category: Network
Repository: net
Title: client init scripts
Maintainer: Findlay Feng <i@fengch.me>
Source: hysteria-2.5.0.tar.zst
License: MIT
URL: https://hysteria.network
Type: ipkg
Description:            Hysteria is a feature-packed proxy & relay tool optimized for lossy,
        unstable connections (e.g. satellite networks, congested public Wi-Fi,
        connecting to foreign servers from China) powered
        by a customized protocol based on QUIC.
@@

Makefile:34: /golang-package.mk: No such file or directory
make[2]: *** No rule to make target '/golang-package.mk'.  Stop.
```

Reply:
我看到错误应该是GO_INCLUDE_DIR没有被定义的原因？
我可以添加
```make
GO_INCLUDE_DIR?=$(TOPDIR)/feeds/packages/lang/golang
```
使用
```make
include ../../lang/golang/golang-package.mk
```
带来的问是，如果包并没有在packages的路径中时，会找不到目标

---
I see the error should be because GO_INCLUDE_DIR is not defined?
I can add
```make
GO_INCLUDE_DIR?=$(TOPDIR)/feeds/packages/lang/golang
```
Use
```make
include ../../lang/golang/golang-package.mk
```
The problem is that if the package is not in the packages path, the target cannot be found

Reply:
Whether it's $(TOPDIR)/feeds/packages/or $(TOPDIR)/packages/feeds, use wildcard characters..// It can be matched with better compatibility. Look at other software packages that do this, such as [kcptun](https://github.com/openwrt/packages/blob/master/net/kcptun/Makefile)

Reply:
> The problem is that if the package is not in the packages path, the target cannot be found

it should be there and it's there.